roles/tendrl-server/tasks/rsyslog.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: ensure rsyslog is enabled and running SERVER-NGCHECKINGCHECKINGCHECKINGCHECKINGCHECKING
```

Comment:
What is this long string?

Reply:
Typo :) I add this line to quickly check the debug message. I will remove this string.